lib/assessment.js

```diff
@@ -4,24 +4,31 @@
 // Write a function that takes zero or more arguments
 // and returns the minimum from that set
 // assume that all the argument are numbers
-var min = function min() {
-
+var min = function min(arg) {
+  min = math.min();
 };
+//I used mdn
 
 // Question 2
 // Write a function that takes zero or more arguments
 // and returns the sum of that set
 // assume that all the argument are numbers
 var sum = function sum() {
-
+  return math.sum();
 };
 
 // Question 3
 // Write a function that returns an array of the specified length
 // with all the elements set to defaultValue
 var newArray = function newArray(length, defaultValue) {
+  var newArray = [defaultValue];
+  for (i=0; i < length.length; i++) {
```

Comment:
Did you mean `newArray.length`? Arrays have a `length` property. Numbers do not.